proxy/proxy/redirect/rhnRedirect.py

```diff
@@ -414,3 +414,13 @@ def __redirectFailover(self):
             status = apache.HTTP_OK
 
         return status
+
+
+def log_debug_facade(level, *args):
+    new_args = ("wsgi_redirect", ) + args
+    log_debug(level, *new_args)
+
+
+def log_error_facade(*args):
+    new_args = ("wsgi_redirect", ) + args
+    log_error(*new_args)
```

Comment:
This is not what I imagined.

What I had in mind is to adapt [rhnLog.py#L79](https://github.com/uyuni-project/uyuni/blob/master/python/spacewalk/common/rhnLog.py#L79) so that we would have:
```python
def initLOG(log_file="stderr", level=0, component=""):
```
this `component` we will store in log object and adapt `logMessage` to prepend `component` to messages.

We then can adapt [apacheServer.py#L47](https://github.com/uyuni-project/uyuni/blob/master/proxy/proxy/apacheServer.py#L47) to

```python
initLOG(CFG.LOG_FILE, CFG.DEBUG, componentType)
```
 call to include the component. And the rest can theoretically remain the same.

Wouldn't that work?

Reply:
Right, it looks like a better solution. I'll try it out